diff --git a/Display.h b/Display.h
index 4b60819..0030a8d 100644
--- a/Display.h
+++ b/Display.h
@@ -135,6 +135,12 @@ void busyCallback(const void* p) { display_callback(); }
   #define SCL_OLED 6
   #define SDA_OLED 5
   #define DISP_CUSTOM_ADDR true
+#elif BOARD_MODEL == BOARD_EORA_S3
+  #define DISP_RST -1
+  #define DISP_ADDR 0x3C
+  #define SCL_OLED 17
+  #define SDA_OLED 18
+  #define DISP_CUSTOM_ADDR true
 #else
   #define DISP_RST -1
   #define DISP_ADDR 0x3C
@@ -379,6 +385,8 @@ bool display_init() {
       Wire.begin(SDA_OLED, SCL_OLED);
     #elif BOARD_MODEL == BOARD_XIAO_S3
       Wire.begin(SDA_OLED, SCL_OLED);
+    #elif BOARD_MODEL == BOARD_EORA_S3
+      Wire.begin(SDA_OLED, SCL_OLED);
     #endif
 
     #if HAS_EEPROM
@@ -490,6 +498,9 @@ bool display_init() {
           #elif BOARD_MODEL == BOARD_TDECK
             disp_mode = DISP_MODE_PORTRAIT;
             display.setRotation(3);
+          #elif BOARD_MODEL == BOARD_EORA_S3
+            disp_mode = DISP_MODE_LANDSCAPE;
+            display.setRotation(0);
           #else
             disp_mode = DISP_MODE_PORTRAIT;
             display.setRotation(3);
