--- a/Makefile
+++ b/Makefile
@@ -128,6 +128,9 @@
 firmware-heltec_w_paper:
 	arduino-cli compile --fqbn esp32:esp32:heltec_wifi_lora_32_V3 $(COMMON_BUILD_FLAGS) --build-property "compiler.cpp.extra_flags=\"-DBOARD_MODEL=0x3E\""
 
+firmware-eora_s3:
+	arduino-cli compile --fqbn esp32:esp32:esp32s3:FlashSize=4M,CDCOnBoot=cdc,USBMode=default $(COMMON_BUILD_FLAGS) --build-property "compiler.cpp.extra_flags=\"-DBOARD_MODEL=0x47\""
+
 firmware-xiao_s3:
 	arduino-cli compile --fqbn esp32:esp32:XIAO_ESP32S3 $(COMMON_BUILD_FLAGS) --build-property "compiler.cpp.extra_flags=\"-DBOARD_MODEL=0x3E\""
 
